{"name": "test_alert2_failure", "status": "passed", "steps": [{"name": "Нажатие кнопки для вызова Alert 2", "status": "passed", "start": 1761746154075, "stop": 1761746154259}, {"name": "Отменить Cancel у Alert 2", "status": "passed", "start": 1761746154263, "stop": 1761746154279}, {"name": "Получить текст после нажатия Alert 2", "status": "passed", "start": 1761746154346, "stop": 1761746154388}, {"name": "Подтверждаем, что после нажатия Отмены у нас выпало нужное предложение", "status": "passed", "start": 1761746154388, "stop": 1761746154388}], "attachments": [{"name": "stdout", "source": "ce55dd71-dc49-46d2-b0b9-9ac45d9f23c7-attachment.txt", "type": "text/plain"}], "start": 1761746154031, "stop": 1761746154389, "uuid": "a188d6f8-74d6-43c2-803c-82ab7b172fca", "historyId": "ff70814ac359c54dad1faef5f68cde8f", "testCaseId": "ff70814ac359c54dad1faef5f68cde8f", "fullName": "test_alerts.TestAlertsPage#test_alert2_failure", "labels": [{"name": "story", "value": "Тест второго Alert (Негативный - нажатие отмены)"}, {"name": "feature", "value": "Alerts"}, {"name": "suite", "value": "test_alerts"}, {"name": "subSuite", "value": "TestAlertsPage"}, {"name": "host", "value": "DESKTOP-T28TT8C"}, {"name": "thread", "value": "27268-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_alerts"}], "titlePath": ["test_alerts.py", "TestAlertsPage"]}